scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -208,6 +207,21 @@ int save_gpu_tgid_pid() {
 	return 0;
 }
 
+SEC("?kprobe/nvidia_open")
+int kprobe_nvidia_open() {
+	return save_gpu_tgid_pid();
+}
+
+SEC("?kprobe/nvidia_poll")
```

Comment:
Is this probe used frequently? If it's on every GPU request it might have some overhead.

Reply:
I can't tell w/ my current test setup (too noisy) but mmap alone works fine.

Reply:
I messed up my test earlier -- no gpu support w/o poll it looks like, adding it back in.